styles.js

```diff
@@ -7,16 +7,27 @@ export default createGlobalStyle`
     box-sizing: border-box;
   }
 
+  
   :root {
     --color-green: #79af6e;
   }
 
+  main {
+    margin-bottom: 15vh;
+  }
```

Comment:
again im not sure, but is think this shouldn't be necessary if the footer and everything is positioned correctly